VehicleInformation/IVehicleInformation.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace AutonomousCars
+{
+    public interface IVehicleInformation
```

Comment:
Why do you need to create an interface for a DTO ?

Reply:
After giving some thought, there is no need for a data transfer object interface. Did not think about it until now.